starteam/starcellbio_html
# replaced soyutils from google with npm soyutils

#### What are the relevant tickets?

fixes #583 
#### What's this PR do?

Removes soyutils.js from Google which, according to TLO, has an incompatible license. 
#### Where should the reviewer start?

@starteam/devops Can you review the changes to the ansible provisioning and travis yml?

@annagav since there are no test for the assignment builder javascript, can you please check to see if the assignment builder is still functional?
#### How should this be manually tested?

Check out a clean repository without any build artifacts. Do a `vagrant up` and run the app. See if the student view and the assignment builder work as expected. 
#### Any background context you want to provide?

There are a few other libraries that need to be removed / replaced as well. See #584. 
#### What GIF best describes this PR or how it makes you feel?

Not a gif, but you get the idea:

![img](http://www.ustrading.com/img/products/xl/20410.jpg)
